docs/mining/overview.md

```diff
@@ -2,7 +2,7 @@
 
 ---
 
-## Overview  
+## DCR PoW Overview  
```

Comment:
This `Overview` heading can be removed completely. We've slowly been removing it from other pages - the fact that the opening paragraph is an overview of the rest of the page is self-evident.

Reply:
Removed